stdlib/universe/cov_test.flux

```diff
@@ -40,7 +40,10 @@ outData =
 "
 
 testcase cov {
-    table = csv.from(csv: inData) |> testing.load()
+    table =
+        csv.from(csv: inData)
+            |> testing.load()
+            |> range(start: 2018-05-22T19:53:00Z, stop: 2018-05-22T19:54:00Z)
```

Comment:
Just an observation: for some of these tests I wonder if we should just remove the call to `testing.load()`. We do not push down `cov()` for example, so I don't see much reason to load data into storage. It would also make the tests run a lot faster.